From cd04ada8f04f618d8f847cec377e2ea18f0b5892 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Wed, 23 Mar 2022 11:19:35 +0200
Subject: [PATCH] [SW-69581] gaudi3: adjust ppboot dccm base addr

ppboot base address is adjusted due to of arc1 memory growth,
and increase in the arcBoot rom dccm consumption.

Change-Id: I453b163019e264e78c16b1abcd2e621ab8d51382
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.h b/arch/gaudi3/arc/common.h
index df5e5ef..a4cb6d4 100644
--- a/arch/gaudi3/arc/common.h
+++ b/arch/gaudi3/arc/common.h
@@ -36,6 +36,7 @@
 #define FW_SRAM_BASE			(FW_MEM_REGION_BASE + 0x3140000)
 #define SP_SRAM_BASE			(FW_MEM_REGION_BASE + 0x3120000)
 #define ARC1_ICCM_BASE			(FW_MEM_REGION_BASE + 0x3040000)
+#define ARC1_DCCM_BASE			(FW_MEM_REGION_BASE + 0x3080000)
 #define ARC0_ICCM_BASE			(FW_MEM_REGION_BASE + 0x3000000)
 #define HL_SPI_BASE			(FW_MEM_REGION_BASE + 0x2000000)
 
@@ -47,10 +48,10 @@
 #define BOOT_INFO_BASE			SP_SRAM_BASE
 
 /* In order not to conflict with ARC BOOTROM's usage of DCCM,
- * Let PPBoot start from the offset of 0x4030, for using DCCM
+ * Let PPBoot start from the offset of 0x5000, for using DCCM
  * for its own BSS and Stack.
  */
-#define PPBOOT_DCCM_BASE		0x23064030
+#define PPBOOT_DCCM_BASE		(ARC1_DCCM_BASE + 0x5000)
 
 #define FUSE_WORDS_PER_BANK		0x80
 #define I2C_SLAVE_ADDRESS		0x6A
